BrainPortal/app/helpers/rich_ui_helper.rb

```diff
@@ -52,6 +52,20 @@ def full_description(description)
     "<span style=\"white-space: pre-wrap;\">".html_safe + description + "</span>".html_safe
   end
 
+  def full_description(description, autolink=false)
```

Comment:
Why do you have two methods with the same name?!?!

Reply:
duplicate is deleted